composer.json

```diff
@@ -17,11 +17,17 @@
         "dhii/php-cs-fixer-config": "dev-php-5.3",
         "phpunit/phpunit": "^4.8",
         "ptrofimov/xpmock": "^1.1",
-        "codeclimate/php-test-reporter": "<=0.3.2"
+        "codeclimate/php-test-reporter": "<=0.3.2",
+        "dhii/validation-interface": "^0.1"
```

Comment:
[This reference to `ValidationFailedExceptionInterface`](https://github.com/Dhii/validation-abstract/blob/develop/src/AbstractValidator.php#L90).

And while they won't break code (due to lack of return type hinting in pre-php-7 syntax), the various `@return` tags also reference interfaces. A couple of examples:

[`AbstractValidator.php#L27`](https://github.com/Dhii/validation-abstract/blob/develop/src/AbstractValidator.php#L27)

[`AbstractValidator.php#L42`](https://github.com/Dhii/validation-abstract/blob/develop/src/AbstractValidator.php#L42)

I can't say if there would be a point in the future where we make this purely PHP 7+ so this could be an irrelevant point for now. However the reference to `ValidationFailedExceptionInterface` should be enough to merit a non-dev dependency.